makefile

```diff
@@ -1,37 +1,53 @@
 FC=gfortran
-FOPT= -O3
+FOPT= -O3 -fPIC #-Wall -fbounds-check
 AUXDIR = ./numerical
 QAGDIR = ./numerical/dqag
+# TSDIR = ./numerical/TSPACK
 
 MAIN = main.o
 MFOBJ = gencap.o
+TRGOBJ = alphakappamod.o transgen.o fastevap.o
 NUMOBJ =  dgamic.o d1mach.o
+AUXOBJ = sgolay.o spline.o pchip.o
```

Comment:
NUMOBJ and AUXOBJ might want to be combined so that all the stuff in numerical is packed together - same for their rules below